website/client/.eslintrc.js

```diff
@@ -14,6 +14,8 @@ module.exports = {
     // TODO find a way to let eslint understand webpack aliases
     'import/no-unresolved': 'off',
     'vue/no-v-html': 'off',
+    'vue/require-prop-types': 'off',
```

Comment:
I would like to keep these ones, we don't respect them often but they're a good practice

Reply:
No worries! If they're good practice we should keep them. I've removed the two "off" lines, and I'll make sure to follow them in the admin panel code.

Would you like me to create an issue for a contributor to fix the code that does't yet follow `vue/require-prop-types` and `vue/require-default-prop`? Perhaps in batches so it's not overwhelming to fix and review all of them at once?

Reply:
Hmm I'd say an issue would be overwhelming, let's keep it like it's and I'll try to follow the convention more closely / remind it to contributors. For your PR I'd say the more important one is the `vue/require-prop-types` and setting `required` to props that are always necessary (it will make  `vue/require-default-prop` not required, it's mandatory only if the prop is not required)